src/components/attendee/VideoHuddle/VideoHuddle.module.scss

```diff
@@ -19,24 +19,35 @@
   }
 
   height: layout.$h-video-chat;
-  padding: space.gap(1);
   border-top: 1px solid theme.$sparkle--border;
   border-bottom: 1px solid theme.$sparkle--border;
-
-  @media (min-width: breakpoint.$md) {
-    border-top: none;
-  }
 }
 
 .VideoHuddleControls {
   @include sparkle.panel--normal;
   position: relative;
   margin-top: space.gap(-1);
-  width: space.empty(1);
   display: flex;
   flex-direction: column;
-  justify-self: flex-start;
+  justify-self: start;
   border-left: 1px solid theme.$sparkle--border;
+
+  span {
+    border-bottom: 1px solid theme.$sparkle--border;
+    width: 32px;
+    height: 32px;
```

Comment:
Temporary numbers until I talk over the details with Sabrina/Matt

Reply:
Private constants to the module `$-size: 32px`, but also, this reminds me we didn't bring `square-size` mixin from the old design.

Reply:
Good idea